src/Stats/TransList.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { ListItem } from 'react-native-elements';
+import GradientIcon from '../Common/icon';
+import styles from '../Common/themeStyle';
+import utils from './utils';
+
+export default function TransList(props) {
+  const { transactions } = props;
+
+  return (
+    transactions.map((value, index) => (
+      <View style={styles.card} key={`DateGroup:${index + 1}`}>
+        <View style={styles.cardAlign}>
+          <Text style={styles.cardHeader}>
+            {value[0].date}
+          </Text>
+          <Text style={styles.cardHeader}>
+            {utils.sumAmount(value)}
+          </Text>
+        </View>
+        {value.map((item, idx) => (
+          <ListItem
+            key={`InfoGroup:${idx + 1}`}
+            leftElement={<GradientIcon name={item.label ? item.label.icon : 'home'} color={item.label && item.label.color ? item.label.color : '#9e87fc'} iconFamily={item.label && item.label.iconFamily ? item.label.iconFamily : ''} />}
+            title={item.label ? item.label.name : 'Unknown'}
+            subtitle={item.comment ? item.comment : 'Unknown'}
+            subtitleStyle={{ color: 'grey' }}
+            rightTitle={item.type === 'Expense' ? `-$${item.amount}` : `+$${item.amount}`}
```

Comment:
we use this `+$/-$` so frequently, shouldnt we make a function for it?

Reply:
Done